projects/plugins/beta/src/admin/exception.template.php

```diff
@@ -2,28 +2,24 @@
 /**
  * Template to display an exception.
  *
+ * @html-template \Automattic\JetpackBeta\Admin::render
+ * @html-template-var Exception $exception
  * @package automattic/jetpack-beta
  */
 
+// phpcs:disable VariableAnalysis.CodeAnalysis.VariableAnalysis.UndefinedVariable -- HTML template, let Phan handle it.
+
 // Check that the file is not accessed directly.
 if ( ! defined( 'ABSPATH' ) ) {
 	exit;
 }
 
-// @global Exception $exception Exception to display.
-if ( ! isset( $exception ) ) {
-	throw new InvalidArgumentException( 'Template parameter $exception missing' );
-}
-$exception = $exception; // Dummy assignment to fool VariableAnalysis.CodeAnalysis.VariableAnalysis.UndefinedVariable.
-
-// -------------
-
 ?>
 <div class="notice notice-error">
 	<p><?php echo esc_html( $exception->getMessage() ); ?></p>
 	<!--
 <?php
-// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped -- We're printing into a comment, and escaping any "--" so it should be impossible to end the comment early.
```

Comment:
Might be worth explicitly mentioning `0x2D` → `0x2212`, as this is confusing at a glance.

Reply:
1b8c20ed1